common/metrics/metric_defs.go

```diff
@@ -1019,7 +1019,7 @@ const (
 	// HistoryDeleteWorkflowExecutionScope tracks DeleteWorkflowExecutions API calls
 	HistoryDeleteWorkflowExecutionScope = "DeleteWorkflowExecution"
 	// HistoryCacheGetOrCreateScope is the scope used by history cache
-	HistoryCacheGetOrCreateScope = "CacheGetOrCreate"
+	HistoryCacheGetOrCreateScope = "HistoryCacheGetOrCreate"
```

Comment:
do we need to update query for this change?

Reply:
HistoryGetOrCreate is the correct one. I pasted the wrong value in previous PR